### PR TITLE
Added resource and attributes creation for the SDK set up

### LINKED ISSUE #1582: Adding attributes to the SDK set up in the Java manual instrumentation
**What needs to be changed?**
The SDK set up in the Java manual instrumentation page is missing to mention attributes to be set to get it working, like for example the service name which is the main one (as it is mentioned in the corresponding agent documentation https://opentelemetry.io/docs/instrumentation/java/automatic/#configuring-the-agent).
We would need to change the code snippet and the text to mention about attributes.

**What is the name + path of the page that needs changed?**
The page is at `content/en/docs/instrumentation/java/manual.md`

### PR DESCRIPTION
**Pull request by ppatierno**
This PR fixes #1582 by adding the resource and attributes creation for the SDK set up.

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
Looks good to me. Will defer to @open-telemetry/java-maintainers on these being the right attributes to add during setup.

File: content/en/docs/instrumentation/java/manual.md
Comment (mateuszrzeszutek) [inline content/en/docs/instrumentation/java/manual.md:34]: It's not really "necessary"/"mandatory", you can create an SDK instance without it and you won't get any error. It's still highly recommended though:
```suggestion
It is also strongly recommended to define a `Resource` instance as a representation of the
entity producing the telemetry; in particular the `service.name` attribute is
the most important piece of telemetry source-identifying info.
```

File: content/en/docs/instrumentation/java/manual.md
Comment (mateuszrzeszutek) [inline content/en/docs/instrumentation/java/manual.md:43]: Let's keep the default attributes set too:

```suggestion
Resource resource = Resource.getDefault()
  .merge(Resource.create(Attributes.of(ResourceAttributes.SERVICE_NAME, "logical-service-name")));
```

File: content/en/docs/instrumentation/java/manual.md
Comment (ppatierno) [reply content/en/docs/instrumentation/java/manual.md:34]: Yeah, it's a bad usage of "mandatory" ... I was thinking that without that attribute you don't have a way to filter in the Jaeger UI AFAIK.

**Review by mateuszrzeszutek — APPROVED** [review]
LGTM 👍 
Thanks!

**Review by chalin — APPROVED** [review]
Thanks all

**Pull request merged**
